drivers/gpu/drm/bridge/sn65dsi84/sn65dsi84.c

```diff
@@ -1582,11 +1600,26 @@ static bool sn65dsi84_videomode_parse_dt(struct device_node *np, struct sn65dsi8
 static int sn65dsi84_parse_dt(struct device_node *np, struct sn65dsi84 *adv)
 {
     struct device_node *endpoint;
-	int ret;
+    struct device_node *i2c_edid_np;
+    int ret;
```

Comment:
Unused variable. Didn't you get a warning from the compiler?

Reply:
Although a compile warning is easily missed in the amount of compile output messages of a Kernel build, this variable is not unused, it is used first at line 1628.